docs/manuals/core/config/job-config.md

```diff
@@ -112,96 +112,16 @@ _airflow_ mode with the following config :
     [JOB]
     mode = "airflow"
     ```
-
-## Starting Airflow from Taipy
-
-To let Taipy start the Airflow service, you can use the following configuration:
-
-=== "Python configuration"
-
-    ```python  linenums="1"
-    from taipy import Config
-
-    Config.configure_job_executions(mode="airflow", start_airflow=True)
-    ```
-
-=== "TOML configuration "
-
-    ```python linenums="1"
-    from taipy import Config
-
-    Config.load("config.toml")
-    ```
-
-    ```toml linenums="1" title="config.toml"
-    [JOB]
-    mode = "airflow"
-    start_airflow = "True:bool"
-    ```
-
-By default, Airflow creates a local folder `.airflow` to store its dependencies.
-You can change this location with the `airflow_folder` config:
-
-=== "Python configuration"
-
-    ```python  linenums="1"
-    from taipy import Config
-
-    Config.configure_job_executions(mode="airflow", airflow_folder="my_custom_path")
-    ```
-
-=== "TOML configuration"
-
-    ```python linenums="1"
-    from taipy import Config
-
-    Config.load("config.toml")
-    ```
-
-    ```toml linenums="1" title="config.toml"
-    [JOB]
-    mode = "airflow"
-    airflow_folder = "my_custom_path"
-    ```
-
-!!! warning "Production setting"
-
-    Taipy starts Airflow in `standalone` mode. It is an Airflow development mode and not recommended for production.
-
 ## Using an external Airflow
```

Comment:
Do we need to keep this sub-section since we are always running with an external airflow?
```suggestion
```

Instead, I would add a small sentence at the beginning of the section 
`# Using Airflow (Enterprise version only)` to explain that the user need to have an external airflow running.